### PR TITLE
git test

### PR DESCRIPTION
**Pull request by rs-sdet**
Chnges

### CODE REVIEW

File: src/main/java/com/ui/pages/LoginPage.java [hunk 1]
```diff
@@ -43,6 +43,7 @@ public void verifyPageHeaders(){
         String title = homePageTitle.getText();
         Assert.assertEquals("Register", title, "Not navigated to the register page");
         utils.navigateToBack();
+        System.out.println("Git Test");
```
Comment (Riyaz-commits) [inline src/main/java/com/ui/pages/LoginPage.java]: can you print the title int eh priting statement

**Review by Riyaz-commits — CHANGES_REQUESTED** [review]

File: src/main/java/com/ui/pages/LoginPage.java (hunk 1 above)
Comment (rs-sdet) [reply src/main/java/com/ui/pages/LoginPage.java]: Changes are made. Please review the code

Comment (Riyaz-commits) [reply src/main/java/com/ui/pages/LoginPage.java]: Looks good

**Review by Riyaz-commits — APPROVED** [review]
LGTM!

**Pull request merged**